inspirehep/modules/literaturesuggest/forms.py

```diff
@@ -62,6 +64,8 @@
 BOOK_CLASS = " book-related"
 PROCEEDINGS_CLASS = " proceedings-related"
 
+FORM_LANGUAGES = ['en', 'ru', 'de', 'fr', 'it', 'es', 'zh', 'pt']
```

Comment:
Looking to: https://github.com/inspirehep/inspire-next/blob/master/inspirehep/dojson/current_marcxml_usage/hep.txt#L1273-L1286 I think you should also add at least `jp` for Japan